src/python/pants/engine/internals/parametrize.py

```diff
@@ -56,7 +56,7 @@ class _MergeBehaviour(Enum):
     merge_behaviour: _MergeBehaviour = dataclasses.field(compare=False)
 
     def __init__(self, *args: str, **kwargs: Any) -> None:
-        object.__setattr__(self, "args", args)
+        object.__setattr__(self, "args", tuple(args))
```

Comment:
was this needed? I would've expected a `*args` to always be a tuple..

Reply:
It was not needed, I must have removed it at some point since it's not in my current branch.